client/src/js/components/bhDateInterval.js

```diff
@@ -19,6 +19,7 @@ angular.module('bhima.components')
       validationTrigger : '<', // validation trigger action
       dateFrom : '=',          // date from
       dateTo : '=',            // date to
+      dateId : '@',            // date identifier
```

Comment:
It is for case where we have multiple `<bh-date-interval>` component in the same view for having a mean to perform tests on them.